tests/validation/tests/v3_api/common.py

```diff
@@ -2311,3 +2315,48 @@ def auth_resource_cleanup():
         user_crtbs = client.list_cluster_role_template_binding(userId=user.id)
         for crtb in user_crtbs:
             client.delete(crtb)
+
+
+class LogParse:
```

Comment:
rename to specify this is specific to websockets

Reply:
Renaming is done.